CSS/main.css

```diff
@@ -0,0 +1,161 @@
+.body {
```

Comment:
There is no need to add class name for body. You can use selector by tag (only for body)

Reply:
I've done it.